tests/test_arithmetic.py

```diff
@@ -0,0 +1,90 @@
+"""
+"""
+import pytest
+
+import calc
+
+
+def test_parse_subtraction():
+    input_text = "5-3"
+    calculator = calc.Calc(text=input_text)
+    assert calculator.parse() == 2
+
+
+def test_parse_multiplication_identity():
```

Comment:
You have multiplication and division identity functions, but no subtraction or addition identity functions. Why?